bufferevent_pair.c

```diff
@@ -316,7 +316,8 @@ be_pair_flush(struct bufferevent *bev, short iotype,
 
 	incref_and_lock(bev);
 
-	partner = downcast(bev_p->partner);
+	if (!(partner = downcast(bev_p->partner)))
+		return -1;
```

Comment:
> - While bev_p->partner is checked, when libevent calls
>`downcast(bev_p->partner)` it dereferences `pair->bev.bev`
> which can trigger a NULL deref in `be_pair_transfer` for
> the variable `partner`.

> Fix: check the return value of `downcast`

Have you seen it? I understand that it is too late, but still, maybe you have backtrace?